pds/server.go

```diff
@@ -33,7 +33,6 @@ import (
 	"github.com/labstack/echo/v4"
 	"github.com/labstack/echo/v4/middleware"
 	"github.com/lestrrat-go/jwx/v2/jwt"
-	"github.com/whyrusleeping/go-did"
```

Comment:
Looks like my editor whipped this import out since it was being used in this file.